src/session.rs

```diff
@@ -221,6 +221,7 @@ impl Session {
             sender_ssrc,
             do_nack,
             &self.medias,
+            &self.codec_config,
```

Comment:
This seems wrong. We need to intersect the session codecs with those negotiated for this media, otherwise we might use an RTX PT that the remote peer has disabled.

Reply:
Yeah, if you look at `Media::first_pt_with_rtx` it does that.

Reply:
Did you just fix that? I could've sworn I checked `Media::first_pt_with_rtx` and the `self.remote_pts.contains(&p.pt)` bit wasn't there

Reply:
The sneaky fox...